### ISSUE TITLE
Why was alphapose chosen?

### ISSUE THREAD

**Issue by YAYAYru** [midusi/LSA-T#7]
Why was alphapose chosen? What about Mediapipe or Openpose?

**Comment by pedroodb** [comment]
Alphapose was chosen as it allowed us to use the Halpe dataset format, which we found useful for sign language representation, and also for performance reasons as it can run on 8GB of RAM.

Probably Mediapipe and Openpose could work as well, but we didn't find any particular advantage.

**Comment by YAYAYru** [comment]
What is the particular special advantage of Halpe dataset format for sign language representation if you could answer? Would you be able to show an example of sign language representation? 

> Halpe dataset format, which we found useful for sign language representation

**Comment by facundoq** [comment]
Err.. it has more finger keypoints than other formats/models. Also
Alphapose was (is?) significantly easier to work with since it is based on
pytorch.

Can I ask why the interest in such a specific detail? Also, you can always
run openpose on the videos...

**Comment by YAYAYru** [comment]
Yes, you can. The main problem with tracked skeletal fingers for sign language recognition task is that I'm interested in a specific detail. I tried OpenPose before, but there is a devops problem with python. Now I use Mediapipe Hands and Pose, Holistic, it's easier to do something there and it's convenient to analyze the finger skeletal model, see [my video example](https://youtu.be/8y2rOph1GYs).

I know more about Mediapipe than Alphapose. Alphapose and Mediapipe hands are the same number - 21 finger key points. I'm asking you, do you use a 2D or 3D finger skeletal model from Alphapose? Any other specific detail?:)
